### PR TITLE
Feat/client side data

### PR DESCRIPTION
**Pull request by JoachimStanislaus**
- Client side functionalities ( Ability to work in tandem )
   - Filter
   - Sort
   - Groupby
   - Aggregate
      - Sum, High, Low, Average, Count, Distinct
      
- Persistent Aggregated Data Display.
- Dynamic Fake Data Generator

### CODE REVIEW

File: toolbox/test-graphviz.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
+digraph regexp {
```
Comment (heswell) [inline toolbox/test-graphviz.txt:1]: can you remove this file from the commit. It's an artifact of the server build. Should be in the gotignore

File: vuu-ui/package.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
     "eslint-config-prettier": "8.7.0",
     "eslint-plugin-cypress": "^2.13.3",
     "eslint-plugin-prettier": "4.2.1",
-    "eslint-plugin-react": "7.32.2",
```
Comment (heswell) [inline vuu-ui/package.json:71]: why this change ?

File: vuu-ui/packages/vuu-data/src/array-data-source/array-data-source.ts [hunk 3]
```diff
@@ -40,6 +44,8 @@ import {
 } from "../vuuUIMessageTypes";
 import { collapseGroup, expandGroup, GroupMap, groupRows } from "./group-utils";
 import { sortRows } from "./sort-utils";
+import { el } from "@faker-js/faker";
```
Comment (heswell) [inline vuu-ui/packages/vuu-data/src/array-data-source/array-data-source.ts:47]: don't think we are using faker in here, are we ?

Comment (JoachimStanislaus) [reply vuu-ui/packages/vuu-data/src/array-data-source/array-data-source.ts:47]: sorry about it! Forgot to change the tag to WIP -- shifting faker to another file to make it more modular

File: vuu-ui/packages/vuu-data/src/array-data-source/array-data-source.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { DataSourceFilter, DataSourceRow } from "@finos/vuu-data-types";
```
Comment (heswell) [inline vuu-ui/packages/vuu-data/src/array-data-source/array-data-source.ts:1]: doesn't stop us merging, but we're still lacking a `set config` that will allow us to set multiple attributes at same time. Will be needed for full compatability with RemoteDataSource

Comment (JoachimStanislaus) [reply vuu-ui/packages/vuu-data/src/array-data-source/array-data-source.ts:1]: Will quickly add it in in the coming week! I shifted off that after discussion with Steve that it would be more modular with the current setup.

File: vuu-ui/package.json (hunk 2 above)
Comment (JoachimStanislaus) [reply vuu-ui/package.json:71]: Think it was an auto dependency update, but I'll switch it back manually next push!
